canonical-websites/maas.io
# External link fix

## Done

Fix external link in webkit

## QA

./run and go to http://0.0.0.0:8006/ and make sure the external icons are showing in Chrome
